File: eth/scripts/deploy_testnet/deploy_test_tokens.py
from brownie import *
import json
import os
from dotenv import load_dotenv, find_dotenv


def main():
    load_dotenv(find_dotenv())
    # Opening json

    print(f"You are using the '{network.show_active()}' network")
    if (network.show_active() == 'development'):
        deployer = accounts[0]
    else:
        deployer = accounts.add(os.getenv("DEPLOYER_PRIVATE_KEY"))

    print(f"You are using: 'deployer' [{deployer.address}]")

    # Decimals are equal to mainnet.
    test_USDC = deployer.deploy(TestToken, 6, "TEST_USDC", "TUSDC", publish_source=True)
    test_WBTC = deployer.deploy(TestToken, 8, "TEST_WBTC", "TWBTC", publish_source=True)
    test_WETH = deployer.deploy(TestToken, 18, "TEST_WETH", "TWETH", publish_source=True)
    
    print(f"TEST USDC at {test_USDC}")
    print(f"TEST WBTC at {test_WBTC}")
    print(f"TEST WETH at {test_WETH}")

File: eth/scripts/deployment/deploy-test-tokens.js
const { ethers, upgrades } = require("hardhat");
const { getAddressSaver } = require("../utilities/helpers");
const path = require("path");
const { test } = require("mocha");

async function main() {
    const [deployer] = await ethers.getSigners();
    const network = (await ethers.getDefaultProvider().getNetwork()).name;
    const addressesPath = path.join(__dirname, "./deploymentAddresses.json");
    const saveAddress = getAddressSaver(addressesPath, network, true);

    const TestToken = (await ethers.getContractFactory("TestToken")).connect(deployer);

    const usdcDecimals = 6;
    const wbtcDecimals = 8;
    const wethDecimals = 18;

    const testUSDC = await TestToken.deploy(usdcDecimals, "TEST_USDC", "TUSDC");
    await testUSDC.deployed();
    const testWBTC = await TestToken.deploy(wbtcDecimals, "TEST_WBTC", "TWBTC");
    await testWBTC.deployed();
    const testWETH = await TestToken.deploy(wethDecimals, "TEST_WETH", "TWETH");
    await testWETH.deployed();

    console.log(`TEST USDC at ${testUSDC.address}`);
    console.log(`TEST WBTC at ${testWBTC.address}`);
    console.log(`TEST WETH at ${testWETH.address}`);
}

main().catch((error) => {
    console.error(error);
    process.exitCode = 1;
});

exports.deployTestToken = main